src/App.css

```diff
@@ -1,8 +1,21 @@
-.App {
+.BangWords {
   text-align: center;
+  background-color: #454d5e;
+  min-height: 100vh;
+}
+
+.BangWords-header {
+  background-color: #282c34;
+  min-height: 15vh;
```

Comment:
I love the functionality and structure so I'm just gonna add a couple of design suggestions: maybe this header is too thick?